.release-please-manifest.json

```diff
@@ -1 +1 @@
-{".":"0.3.1"}
+{ ".": "0.3.2" }
```

Comment:
This will be changed by release bot as part of the release PR [like this](https://github.com/noir-lang/noir_rsa/pull/22) right? Release PR should be automatically created once this PR is merged to main

Reply:
Oh I missed this when reviewing, I do expect this to be done by the release bot. The bot should create a release PR on its own.

Reply:
We should also rename the PR title to start with _feat:_ ?
chores don't trigger a release PR

Reply:
Oh didn't realize, will change it